docs/install.md

```diff
@@ -842,8 +842,9 @@ The current API for PMEM-CSI `Deployment` resources is:
 | nodeControllerKey | string | Encoded RSA private key used for signing by `nodeControllerCert` | generated by the operator |
 | caCert | string | Certificate of the CA by which the `registryCert` and `controllerCert` are signed | self-signed certificate generated by the operator |
 | nodeSelector | string map | [Labels to use for selecting Nodes](../docs/install.md#run-pmem-csi-on-kubernetes) on which PMEM-CSI driver should run. | `{ "storage": "pmem" }`|
-| pmemPercentage | integer | Percentage of PMEM space to be used by the driver on each node. This is only valid for a driver deployed in `lvm` mode. This field can be modified, but by that time the old value may have been used already. Reducing the percentage is not supported. | 100 |
-| labels | string map | Additional labels for all objects created by the operator. Can be modified after the initial creation, but removed labels will not be removed from existing objects because the operator cannot know which labels it needs to remove and which it has to leave in place. |
+| pmemPercentage | integer | Percentage of PMEM space to be used by the driver on each node. This is only valid for a driver deployed in `lvm` mode. This field can be modified, but by that time the old value may have been used already. Reducing the percentage is not supported | 100 |
+| labels | string map | Additional labels for all objects created by the operator. Can be modified after the initial creation, but removed labels will not be removed from existing objects because the operator cannot know which labels it needs to remove and which it has to leave in place |
```

Comment:
Why are you removing the trailng dot? A full sentence ends there, so IMHO it's appropriate.

Reply:
Umm, I am not sure, but somewhere in my mind that you mentioned in one of your earlier reviews that sentences in a table(or list?) entry should not end with a dot. And observed other table entries are not ending with a dot. Otherwise, there is no reason to remove them :) I will revert.

Reply:
Reverted this change.